webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const { merge } = require('webpack-merge');
+const {loadDev} = require("./webpack/dev")
+const {common} = require("./webpack/prod")
```

Comment:
Если этой общий конфиг, то лучше его так и назвать 
```suggestion
const {common} = require("./webpack/common")
```